app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,85 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+    erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if logged_in?
+      if params[:content] == ""
+        redirect to '/tweets/new'
+      else
+        @tweet = current_user.tweets.build(content: params[:content])
```

Comment:
Good job making use of the `has_many` relationship on User! And I don't often see students using `build` at this point in the curriculum.